docs/android/platform/binding-java-library/customizing-bindings/index.md

```diff
@@ -48,7 +46,3 @@ explain how to modify this metadata to address these issues:
 
 - [Naming Parameters with Javadoc](~/android/platform/binding-java-library/customizing-bindings/naming-parameters-with-javadoc.md)
     explains how to recover parameter names in a Java Binding Project by using Javadoc produced from the bound Java project.
-
```

Comment:
@nschonni is this introducing `MD047/single-trailing-newline: Files should end with a single newline character` ?

Reply:
MD047 is already enabled, these would just be removing multiple trailing spaces. AFAIK, 47 just looks to see if there is at least one trailing space